feature_engine/feature_selection.py

```diff
@@ -0,0 +1,85 @@
+
+import warnings
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+from feature_engine.dataframe_checks import _is_dataframe, _check_input_matches_training_df
+from feature_engine.variable_manipulation import _define_variables
+
+
+class DropFeatures(BaseEstimator, TransformerMixin):
+    """
+    The FeatureEliminator() drops the list of variable(s) as provided by the user
+    from the dataframe and returns the subset of original dataframe with remaining
+    variables.
+
+    Parameters
+    ----------
+
+    features_to_drop : str or list, default=None
+        Desired variable/s to be dropped from the dataframe
+
+    """
+
+    def __init__(self, features_to_drop=None):
+        self.features = _define_variables(features_to_drop)
+
+    def fit(self, X, y=None):
+        """
+        Verifies that the passed input X if of the type pandas dataframe
```

Comment:
Verifies that the input X **is** a pandas dataframe

could we change for the above?